tests/languages/jexl/operator_feature.test

```diff
@@ -0,0 +1,33 @@
+-
++
+< <=
+> >=
+= ==

```

Comment:
Is `=` even an operator? The doc doesn't mention it.

Reply:
Good catch! Removed it.